account_tax_balance/models/account_tax.py

```diff
@@ -152,16 +153,15 @@ def get_move_lines_domain(self, tax_or_base='tax', move_type=None):
             balance_domain = self.get_base_balance_domain(
                 state_list, type_list)
         domain.extend(balance_domain)
-        return move_line_model.search(domain)
+        return domain
 
     def get_lines_action(self, tax_or_base='tax', move_type=None):
-        move_lines = self.get_move_lines_domain(
+        domain = self.get_move_lines_domain(
             tax_or_base=tax_or_base, move_type=move_type)
-        move_line_ids = [l.id for l in move_lines]
         action = self.env.ref('account.action_account_moves_all_tree')
         vals = action.read()[0]
         vals['context'] = {}
-        vals['domain'] = [('id', 'in', move_line_ids)]
+        vals['domain'] = domain
```

Comment:
I'm quite sure it's better to put the domain in the action, otherwise you move around millions of ids and you kill the browser and the database.